config/Common.php

```diff
@@ -40,7 +40,7 @@ public function modify(Container $di): void
         $dispatcher = $di->get('aura/cli-kernel:dispatcher');
         $dispatcher->setObject(
             'help',
-            $di->newInstance('Aura\Cli_Kernel\HelpCommand')
+            $di->lazyNew('Aura\Cli_Kernel\HelpCommand')
```

Comment:
Hey @kenjis , 

I looked into this. This change is the breaking one. The Aura.Di on 2.x will also will be having the same issue if you try it. Also the Aura.Dispatcher is setting an object. Not a lazy instance. I was trying something like this.

```php
<?php
namespace Aura\Cli_Kernel;

use Aura\Dispatcher\Dispatcher;
use Aura\Di\Injection\LazyInterface;

class CliDispatcher extends Dispatcher
{

    protected function dispatch($object, $params = [])
    {
        if ($object instanceof LazyInterface) {
            $object = get instance
        }

        parent::dispatch($object, $params);
    }

}
```

But I don't know if we want to go that route. Because the `setObject` is conflicting the name. May be we need as different name like setLazyObject . Too much of configuration and that will confuse people.

For now probably we can drop this PR.